backends-clickhouse/src/main/scala/io/glutenproject/execution/CHFilterExecTransformer.scala

```diff
@@ -75,7 +75,9 @@ case class CHFilterExecTransformer(condition: Expression, child: SparkPlan)
     if (leftCondition == null) {
       // The computing for this filter is not needed.
       context.registerEmptyRelToOperator(operatorId)
-      return childCtx
+      // Since some columns' nullability will be removed after this filter, we need to update the
+      // outputAttributes of child context.
+      TransformContext(childCtx.inputAttributes, output, childCtx.root)
```

Comment:
maybe can not return the output of the filter when all the filter are pushdowned, some attrs may be removed in this case. @rui-mo

Reply:
In this case, this `fiter` is no sense.

Reply:
In Velox backend, if all the conditions are pushed down, the filter's computing is ignored.